Src/PerformanceCollector/Perf.Shared/Implementation/QuickPulse/Helpers/QuickPulseThreadModuleScheduler.cs

```diff
@@ -47,7 +48,23 @@ public void Dispose()
 
             private void Worker(object state)
             {
-                (state as Action<CancellationToken>)?.Invoke(this.cancellationTokenSource.Token);
+                try
+                {
+                    (state as Action<CancellationToken>)?.Invoke(this.cancellationTokenSource.Token);
+                }
+                catch(Exception ex)
+                {
+                    // This is a Thread, and we don't want any exception thrown ever from this part as this would cause application crash.
+                    try
```

Comment:
from what  i found, there is another place like this https://github.com/microsoft/ApplicationInsights-dotnet-server/blob/develop/Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryModule.cs#L310

Reply:
and here 

https://github.com/microsoft/ApplicationInsights-dotnet-server/blob/develop/Src/PerformanceCollector/Perf.Shared/QuickPulseTelemetryModule.cs#L520

Reply:
As discussed offline, all these are run inside same thread worker where we try--catch everything,